packages/styles/scss/ibm-dotcom-styles.scss

```diff
@@ -27,3 +27,4 @@
 @import 'patterns/leadspace/leadspace';
 @import 'patterns/listsection/listsection';
 @import 'patterns/simplelongform/simplelongform';
+@import 'patters/pictogramarray/pictogramarray';
```

Comment:
@RaphaelAmadeu misspelled `patterns` folder. Also more nitpicking, can the imports be in alphabetical order?

Reply:
I've ordered it properly in the last commit.